requirements.txt

```diff
@@ -17,3 +17,4 @@ requests
 sentry-sdk
 siphon
 parse
+nbsphinx # needed for PyWPS 4.2.9 only
```

Comment:
Sorry for this !  PR https://github.com/geopython/pywps/pull/565 should fix it.

Reply:
Ok, so we wait for 4.2.10 on conda then remove this ?

Reply:
Up to you.  If no rush, then wait for `4.2.10` and remove this so we do not forget.  Else we can merge this as-is right now and perform the clean up later.